src/css/Reviews.css

```diff
@@ -0,0 +1,29 @@
+.reviewsContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.reviewsContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    padding: 32px;
+    background-color: #9AD4D6;
+}
+
+.reviewsTitle {
+    margin-bottom: 32px;
+}
+
+
+.vectorShape {
```

Comment:
I had a hard time keeping these vector shapes in the right place! I got some advice to put them in their own parent container, and make the container `position: relative` and the shape `position: absolute`. (So the foreground shape is positioned somewhere precisely within its parent container).